ci/workflows/main-cron.yml

```diff
@@ -655,40 +655,40 @@ steps:
     timeout_in_minutes: 5
 
   # Backwards compatibility tests
-  - label: "Backwards compatibility tests version_offset={{matrix.version_offset}}"
-    key: "backwards-compat-tests"
-    command: "VERSION_OFFSET={{matrix.version_offset}} RW_COMMIT=$BUILDKITE_COMMIT ci/scripts/backwards-compat-test.sh -p ci-release"
-    if: |
-      !(build.pull_request.labels includes "ci/main-cron/run-selected") && build.env("CI_STEPS") == null
-      || build.pull_request.labels includes "ci/run-backwards-compat-tests"
-      || build.env("CI_STEPS") =~ /(^|,)backwards?-compat-tests?(,|$$)/
-    depends_on:
-      - "build"
-    plugins:
-      - docker-compose#v5.1.0:
-          run: source-test-env
-          config: ci/docker-compose.yml
-          mount-buildkite-agent: true
-          environment:
-            - BUILDKITE_BRANCH
-      - ./ci/plugins/upload-failure-logs
-    matrix:
-      setup:
-        # Test the 4 latest versions against the latest main.
-        # e.g.
-        # 1: 2.0.0
-        # 2: 1.1.1
-        # 3: 1.0.1
-        # 4: 1.0.0
-        # It is ordered by the full version number, rather than minor / major version.
-        # We can change to just be on major version in the future.
-        version_offset:
-          - "1"
-          - "2"
-          - "3"
-          - "4"
-    timeout_in_minutes: 30
-    retry: *auto-retry
+  #  - label: "Backwards compatibility tests version_offset={{matrix.version_offset}}"
```

Comment:
I suppose that we will refactor the backup & restore and re-enable the backwards compatibility tests, right? cc. @zwang28

Reply:
Backwards compatibility test is not related to backup & restore.

Backup & restore has been testing with both etcd and sql for some time.

Reply:
2.0 -> 2.1 requires migration script @yezizp2012 , not backup & restore.

Reply:
Currently backward compatibility tests only works for etcd backend, I will enable it once some refactoring is complete.